v2/api/authorization/v1alpha1api20200801preview/role_assignment_types_gen.go

```diff
@@ -239,7 +239,16 @@ func (assignment *RoleAssignment) updateValidations() []func(old runtime.Object)
 		func(old runtime.Object) error {
 			return assignment.validateResourceReferences()
 		},
+		assignment.validateImmutableProperties}
```

Comment:
I think this file isn't gofmt'ed? We might need to fix the `astbuilder` bit that builds this list because I think the `{` should be on the next line? Or maybe I'm crazy.

Reply:
Its the `astbuilder` .. I'll double check and fix it